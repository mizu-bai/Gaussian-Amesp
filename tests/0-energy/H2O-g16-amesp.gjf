%nproc=1
%chk=H2O-g16-amesp.chk
#p external="python3 -u ../../src/gamesp.py" nosymm

H2O energy task (g16-amesp)

0 1
 O                  0.00000000   -0.00000000   -0.11085125
 H                  0.00000000   -0.78383672    0.44340501
 H                 -0.00000000    0.78383672    0.44340501

