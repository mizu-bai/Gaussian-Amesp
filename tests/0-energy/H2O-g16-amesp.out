 Entering Gaussian System, Link 0=g16
 Initial command:
 /opt/g16/l1.exe "/home/jh-li/scratch/g16/Gau-353616.inp" -scrdir="/home/jh-li/scratch/g16/"
 Entering Link 1 = /opt/g16/l1.exe PID=    353619.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                12-Oct-2023 
 ******************************************
 %nproc=1
 Will use up to    1 processors via shared memory.
 %chk=H2O-g16-amesp.chk
 ---------------------------------------------------
 #p external="python3 -u ../../src/gamesp.py" nosymm
 ---------------------------------------------------
 1/38=1,172=1/1;
 2/12=2,15=1,17=6,18=5,40=1/2;
 3/11=9,25=1,30=1,41=9900000,43=2/1;
 4/20=17,24=3,113=1,114=1/2;
 6/7=2,8=2,9=2,10=2/1;
 99/5=1,9=1/99;
 Leave Link    1 at Thu Oct 12 10:24:09 2023, MaxMem=           0 cpu:               0.0 elap:               0.0
 (Enter /opt/g16/l101.exe)
 ---------------------------
 H2O energy task (g16-amesp)
 ---------------------------
 Symbolic Z-matrix:
 Charge =  0 Multiplicity = 1
 O                     0.        0.       -0.11085 
 H                     0.       -0.78384   0.44341 
 H                     0.        0.78384   0.44341 
 
 ITRead=  0  0  0
 MicOpt= -1 -1 -1
 NAtoms=      3 NQM=        3 NQMF=       0 NMMI=      0 NMMIF=      0
                NMic=       0 NMicF=      0.
                    Isotopes and Nuclear Properties:
 (Nuclear quadrupole moments (NQMom) in fm**2, nuclear magnetic moments (NMagM)
  in nuclear magnetons)

  Atom         1           2           3
 IAtWgt=          16           1           1
 AtmWgt=  15.9949146   1.0078250   1.0078250
 NucSpn=           0           1           1
 AtZEff=  -0.0000000  -0.0000000  -0.0000000
 NQMom=    0.0000000   0.0000000   0.0000000
 NMagM=    0.0000000   2.7928460   2.7928460
 AtZNuc=   8.0000000   1.0000000   1.0000000
 Leave Link  101 at Thu Oct 12 10:24:09 2023, MaxMem=   104857600 cpu:               0.1 elap:               0.1
 (Enter /opt/g16/l202.exe)
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          8           0        0.000000    0.000000   -0.110851
      2          1           0        0.000000   -0.783837    0.443405
      3          1           0        0.000000    0.783837    0.443405
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3
     1  O    0.000000
     2  H    0.960000   0.000000
     3  H    0.960000   1.567673   0.000000
 Symmetry turned off by external request.
 Stoichiometry    H2O
 Framework group  C2V[C2(O),SGV(H2)]
 Deg. of freedom     2
 Full point group                 C2V     NOp   4
 Rotational constants (GHZ):         919.0227249         408.0852063         282.5991538
 Leave Link  202 at Thu Oct 12 10:24:09 2023, MaxMem=   104857600 cpu:               0.0 elap:               0.0
 (Enter /opt/g16/l301.exe)
 Standard basis: STO-3G (5D, 7F)
 Ernie: Thresh=  0.10000D-02 Tol=  0.10000D-05 Strict=F.
     7 basis functions,    21 primitive gaussians,     7 cartesian basis functions
     5 alpha electrons        5 beta electrons
       nuclear repulsion energy         9.1571759236 Hartrees.
 IExCor=    0 DFT=F Ex=HF Corr=None ExCW=0 ScaHFX=  1.000000
 ScaDFX=  1.000000  1.000000  1.000000  1.000000 ScalE2=  1.000000  1.000000
 IRadAn=      5 IRanWt=     -1 IRanGd=            0 ICorTp=0 IEmpDi=  4
 NAtoms=    3 NActive=    3 NUniq=    3 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 1 integral format.
 Two-electron integral symmetry is turned off.
 Leave Link  301 at Thu Oct 12 10:24:09 2023, MaxMem=   104857600 cpu:               0.0 elap:               0.0
 (Enter /opt/g16/l402.exe)
 External calculation of energy.
 Running external command "python3 -u ../../src/gamesp.py R"
         input file       "/home/jh-li/scratch/g16/Gau-353619.EIn"
         output file      "/home/jh-li/scratch/g16/Gau-353619.EOu"
         message file     "/home/jh-li/scratch/g16/Gau-353619.EMs"
         fchk file        "/home/jh-li/scratch/g16/Gau-353619.EFC"
         mat. el file     "/home/jh-li/scratch/g16/Gau-353619.EUF"
>>> Running Amesp...
>>> Parsing Amesp output...
>>> 大师大法好！
 Recovered energy= -76.4179972776     dipole=     -0.000000000000     -0.000000000000      1.948200000000
 Energy=    -76.4179973     NIter=   0.
 Dipole moment=      -0.000000      -0.000000       1.948200
 Leave Link  402 at Thu Oct 12 10:24:10 2023, MaxMem=   104857600 cpu:               0.0 elap:               0.7
 (Enter /opt/g16/l601.exe)
 Leave Link  601 at Thu Oct 12 10:24:10 2023, MaxMem=   104857600 cpu:               0.0 elap:               0.0
 (Enter /opt/g16/l9999.exe)
 Unable to Open any file for archive entry.
 1\1\GINC-DESKTOP-1D9QDE1\SP\External='python3 -u ../../src/gamesp.py'\
 ZDO\H2O1\JH-LI\12-Oct-2023\0\\#p external="python3 -u ../../src/gamesp
 .py" nosymm\\H2O energy task (g16-amesp)\\0,1\O,0,0.,0.,-0.11085125\H,
 0,0.,-0.78383672,0.44340501\H,0,0.,0.78383672,0.44340501\\Version=ES64
 L-G16RevC.01\HF=-76.4179973\RMSD=0.000e+00\Dipole=0.,0.,1.9482\PG=C02V
  [C2(O1),SGV(H2)]\\@
 The archive entry for this job was punched.


 HONESTY IN A LAWYER IS LIKE A HEN'S HIND LEGS.
  -- MAGNUS OLESON, LAKE WOBEGON PATRIARCH, C.1875
 Job cpu time:       0 days  0 hours  0 minutes  0.3 seconds.
 Elapsed time:       0 days  0 hours  0 minutes  0.9 seconds.
 File lengths (MBytes):  RWF=      6 Int=      0 D2E=      0 Chk=      1 Scr=      1
 Normal termination of Gaussian 16 at Thu Oct 12 10:24:10 2023.
